lib/index.js

```diff
@@ -72,7 +86,7 @@ AppsFlyer.prototype.track = function(track, fn) {
   var props = reject(track.properties({ revenue: 'af_revenue' }));
   payload.eventValue = isEmpty(props) ? '' : stringify(props);
 
-  var ios = contains(track.library(), 'ios');
+  var ios = deviceType === 'ios';
```

Comment:
why are we overwriting `deviceType` here as well? Also since this has changed from the `contains` to a hardcoded value, wouldn't the first `if` block always pass?